packages/components/psammead-podcast-promo/src/components/card.jsx

```diff
@@ -1,11 +1,12 @@
 import styled from '@emotion/styled';
-import { C_GHOST } from '@bbc/psammead-styles/colours';
+import { C_GHOST, C_EBON } from '@bbc/psammead-styles/colours';
 import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 
 const Card = styled.div`
   position: relative;
   background-color: ${C_GHOST};
   display: flex;
+  box-shadow: 0 0 5px 5px ${C_EBON}08;
```

Comment:
Is the  `08` at the end of this line a typo?

Reply:
Nah that's adding transparency of 0.03 to the `C_EBON` hash value - it's intentional 👍

Reply:
ooh I didn't that was a thing. Thought you'd have to rgba-ify the hex value

Reply:
It's disgustingly unintuitive - flipping between letters & numbers  https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Reply:
If 22% opacity is `38` then what's 23%?
That's right kids!
It's `3B `
OK, now what's 24%?
Yep you guessed it!
`3D`!

Reply:
Oh and uh - because that article just freaked me out over where the AA value goes,

> On web, the alpha hex comes after the 6-char hex.
> On iOS and Android, the alpha hex comes before the 6-char hex.